### ISSUE TITLE
nvidia egpu not showing utilization properly

### ISSUE THREAD

**Issue by sbe-arg** [nokyan/resources#412]
### Is there an existing issue for this?

- [X] I searched the existing issues and did not find anything similar.

### Current Behavior

usage remains 1%

### Expected Behavior

show utilization?

### Environment

```markdown
1.7.0 flatpak

ubuntu 24.04
```

### Anything else?

![image](https://github.com/user-attachments/assets/56ddb597-b905-4b29-943d-a36ffc5f8060)

**Comment by nokyan** [comment]
Hi, thanks for the issue.
Did it work on Resources 1.6?

**Comment by nokyan** [comment]
Do you mind testing the `nvidia-usage-fix` branch to see whether the issue is fixed?

**Comment by sbe-arg** [comment]
How do I test that branch? Need to do the vscode with the extension approach?

**Comment by nokyan** [comment]
> How do I test that branch? Need to do the vscode with the extension approach?

You don't need VSCode, if you want to you can do it all in your terminal. You need flatpak-builder installed, it should be in the Ubuntu repos.
```sh
git clone https://github.com/nokyan/resources
cd resources
git checkout nvidia-usage-fix
flatpak install org.gnome.Sdk//47 org.freedesktop.Sdk.Extension.rust-stable//24.08 org.gnome.Platform//47 org.freedesktop.Sdk.Extension.llvm18//24.08
flatpak-builder --user flatpak_app build-aux/net.nokyan.Resources.Devel.json
flatpak-builder --run flatpak_app build-aux/net.nokyan.Resources.Devel.json resources
```

**Comment by MythicByte** [comment]
I have started some digging in the NVIDIA code for usage and have a question. Why do we calculate the usage of the GPU and then divide it by 100? It makes no sense. I have tested the code on my NVIDIA card, and it is consistently near 0%.

The function in nvidia.rs is as follows:

fn usage(&self) -> Result<f64> {
    Self::nvml_device(&self.pci_slot_string)
        .and_then(|dev| {
            dev.utilization_rates()
                .context("unable to get utilization rates through NVML")
        })
        .map(|usage| usage.gpu as f64 / 100.0)
        .or_else(|_| self.drm_usage().map(|usage| usage as f64 / 100.0))
}
Why do we use .map(|usage| usage.gpu as f64 / 100.0)? I understand the or_else part is just a fallback. I have written a short program to test this, and it makes no sense. 
[Screencast From 2024-12-05 02-06-10.webm](https://github.com/user-attachments/assets/06dac7bb-5e3f-4f18-82e8-fce5393df586)
@nokyan
